templates/index.ts.hbs

```diff
@@ -3,7 +3,11 @@ export { {{apiName}} } from "./{{directoryName}}";
 import { {{apiName}}Types } from "./{{directoryName}}";
 export type { {{apiName}}Types };
 {{/each}}
-export type { ClientConfigInit } from "./clientConfig";
+export type {
+    ClientConfigInit,
+    FetchFunction,
+    FetchOptions
```

Comment:
I'm guessing these were just exports that we missed?

Reply:
Yeah we just never properly exported these types to consume

And unsure if we'll have to do something similar for the node SDK yet since it has a different build process, I'll investigate once I smoke test it